index.js

```diff
@@ -0,0 +1,29 @@
+function calculate(arr) {
+  let isOperand = ['+', '-', '*', '/']
+  // eslint-disable-next-line no-sparse-arrays
+  let invalidInputs = ['', , undefined, null]
+  let validInputs = []
+
+  for (let i = 0; arr.length > i; i++) {
+    if (isOperand.includes(arr[i])) { validInputs.push(arr[i]) }
+    if (!isNaN(arr[i]) && arr[i] !== null && !invalidInputs.includes(arr[i])) { validInputs.push(arr[i]) }
```

Comment:
_Nit_ for long conditional statements like this please break open your brackets onto multiple lines.

```
if (!isNaN(arr[i]) && arr[i] !== null && !invalidInputs.includes(arr[i])) { 
      validInputs.push(arr[i]) 
    }

```